includes/admin/class-wcmp-export.php

```diff
@@ -686,8 +685,14 @@ public static function getRecipientFromOrder(WC_Order $order)
 
                     $address_intl["street"]        = (string) $address_parts["street"];
                     $address_intl["number"]        = (string) $address_parts["number"];
-                    $address_intl["number_suffix"] = array_key_exists("number_suffix", $address_parts) // optional
-                        ? (string) $address_parts["number_suffix"] : "";
+                    $address_intl["number_suffix"] = (string) $address_parts["extension"] ?: "";
+
+                    if (!$address_intl["number_suffix"]) {
+                       if (preg_match("/^[a-zA-Z]$/", $address["street_additional_info"])) {
```

Comment:
Waar komt die regex vandaan en wat moet het matchen? Je kunt hem korter schrijven: `/^[A-z]$/`, maar ik ben wel benieuwd wat hij moet doen. We hebben namelijk al een hele hoop betrouwbare adres regexes :)